src/main/java/cam72cam/immersiverailroading/entity/EntityRidableRollingStock.java

```diff
@@ -140,6 +145,7 @@ public boolean shouldRiderSit() {
 	}
 
 	public Map<UUID, Vec3d> passengerPositions = new HashMap<UUID, Vec3d>();
+	public Map<UUID, Vec3d> passengerDismountAttempts = new HashMap<UUID, Vec3d>();
```

Comment:
This should probably not be a static variable

Reply:
It's static?